dev-docs/RFCs/vNext/layer-shader-customization-rfc.md

```diff
@@ -0,0 +1,113 @@
+# RFC: Layer Shader Customization
+
+* **Authors**: Ib Green, Xiaoji Chen
+* **Date**: Feb 2018
+* **Status**: Draft
+
+## Motivation
```

Comment:
Recently I have been  adding an "Abstract" section that summarizes that actual proposal, before the motivation. Otherwise one has to read quite a bit to understand what is actually being proposed in an RFC. E.g.:

```
##  Abstract 
This RFC defines a new common API for 32 and 64 bit projection. It proposes implementing this interface in the current `project64` shader module, as well as in a new `project32` shader module. ...
```